tests/pactHelper.js

```diff
@@ -108,10 +108,22 @@ const webdavExceptionResponseBody = (exception, message) => new XmlBuilder('1.0'
       .appendElement('s:message', '', message)
   })
 
-const webdavPath = resource => MatchersV3.regex(
-  '.*\\/remote\\.php\\/webdav\\/' + webdavMatcherForResource(resource),
-  `/remote.php/webdav/${resource}`
-)
+const webdavPath = (resource, version = 'old') => {
```

Comment:
Do I get it right, that you could remove the versioning here entirely? Looks like a leftover from the development of this PR. Matchers below are identical.